### PR TITLE
Add doc section on running unit tests

### PR DESCRIPTION
**Pull request by amberarrow**
Added section on running unit tests and installing winutils.exe on Windows.
@tweise Please review and merge

### CODE REVIEW

File: docs/apex_development_setup.md [hunk 1]
```diff
@@ -94,6 +94,38 @@ When the run completes successfully, you should see a new directory named `myape
...
+property `hadoop.home.dir`, for example:
+
+    mvn -Dhadoop.home.dir=c:\hadoop\Software\hadoop-common-2.2.0-bin-master test
+
+or set the environment variable separately:
+
+    set HADOOP_HOME=c:\hadoop\Software\hadoop-common-2.2.0-bin-master
+    mvn test
+
+To run unit tests from within your IDE, set the `HADOOP_HOME` environment
```
Comment (tweise) [inline docs/apex_development_setup.md]: Can also be set system wide and then no IDE tweaks are needed.

Comment (tweise) [inline docs/apex_development_setup.md]: Should mention that first as it covers command line as well.

Comment (amberarrow) [inline docs/apex_development_setup.md]: Fixed

File: docs/apex_development_setup.md [hunk 2]
```diff
@@ -94,6 +94,42 @@ When the run completes successfully, you should see a new directory named `myape
...
 
+Running Unit Tests
+----
+To run unit tests on Linux or OSX, simply run the usual maven command, for example: `mvn test`.
+
+On Windows, an additional file, `winutils.exe`, is required; download it from
+<https://github.com/srccodes/hadoop-common-2.2.0-bin/archive/master.zip>
+and unpack the archive to, say, `C:\hadoop`; this file should be present under
+`hadoop-common-2.2.0-bin-master\bin` within it.
+
```
Comment (sandeshh) [inline docs/apex_development_setup.md:105]: Have we tried this ? In the users mailing list, it was mentioned that 2.6 worked. That doc you referenced is from Windows XP page, which is not supported anymore.

Comment (amberarrow) [inline docs/apex_development_setup.md:105]: Yes, I tried downloading master.zip, unpacking and setting the environment variable locally within the IDE as well as on the console on Windows 10.

Describing how to set the environment variable globally is different in different versions of Windows and covering all variants would introduce too much verbiage into the document. There are tons of web pages that show how to do it, I just picked one.

**Pull request merged**